src/pages/questions/index.jsx

```diff
@@ -18,6 +18,15 @@ export default function Questions() {
     useQuestionBuffer(filterState);
   const question = buffer[0] ?? null;
 
+  const resetFilters = React.useCallback(
```

Comment:
useCallback (memoization) may not be needed in this case, maybe go with a normal arrow function?

Reply:
The `setFilterState` is not usefull in the array of dependencies, because state setter are guaranteed to stay constant between render

It's not entirely useless, because `resetFilters` is passed to `QuestionDisplay`. WIthout that useCallback each render will create a new function, and so `<QuestionDisplay/>` will rerender too

Reply:
Yeah, I guess that the baseline is if the useCallback memoization improves performance. I think that in this case as the QuestionDisplay is a presentational component and doesn't have heavy computations or something really slow, there is not a lot of benefit in memoizing the reset function and makes the code a bit more complex. Also, the memoization is not cost free, it will store in memory and every render needs to compare with dependency array. Besides that, the QuestionDisplay will rerender every time the Questions (index) rerenders, unless QuestionDisplay itself is memoized.